translator/src/test/java/com/google/devtools/j2objc/gen/StatementGeneratorTest.java

```diff
@@ -132,6 +132,20 @@ public void testCatchTranslation() throws IOException {
     assertEquals("@try {\n;\n}\n @catch (JavaLangException *e) {\n}", result);
   }
 
+  public void testMultiCatchTranslation() throws IOException {
```

Comment:
Could you instead merge this into testMultiCatch() below, and use assertTranslatedLines to verify the translation.